experimental/bindings/java/build.gradle

```diff
@@ -25,21 +29,17 @@ android {
 
         externalNativeBuild {
             cmake {
-                arguments '-DIREE_BUILD_EXPERIMENTAL_JAVA_BINDINGS=ON',
-                        '-DIREE_HAL_DRIVERS_TO_BUILD=VMLA',
+                arguments "-DIREE_BUILD_EXPERIMENTAL_JAVA_BINDINGS=ON",
+                        "-DIREE_HAL_DRIVERS_TO_BUILD=VMLA",
```

Comment:
We can also limit which components are built for the host tools with

```
        "-DIREE_TARGET_BACKENDS_TO_BUILD=vmla",
        "-DIREE_HAL_DRIVERS_TO_BUILD=vmla",
```

to match this. That will speed up builds significantly.

In full generality we should build all supported compiler target / drivers combinations (which would exclude Metal, CUDA, etc. on Android). For now the Java bindings only support VMLA (pending a way to create a device/driver of a specific type).